src/backend/dx12/src/lib.rs

```diff
@@ -26,7 +26,11 @@ mod window;
 
 use core::{memory, Features, Limits, QueueType};
 use spirv_cross::hlsl;
+
+#[cfg(not(feature = "copy"))]
 use wio::com::ComPtr;
+#[cfg(feature = "copy")]
+use core::copy::Pointer as ComPtr;
```

Comment:
An issue I see with this approach is the missing `Release` calls once a `ComPtr` drops.